---
name: Samuel Chagas de Assis
date: julho, 2024
qualifications: BCom(Hons), BSc
address: Campinas, São Paulo - Brazil
email: samuel.achagas98@gmail.com
www: chagas98.github.io
github: chagas98
linkedin: samuelchagass
position: Biotechnologist - Computational Biologist
headcolor: 2983f3
output:
  vitae::awesomecv:
    page_total: yes
bibliography: {}
nocite: '@'
csl: /home/samuel/R/x86_64-pc-linux-gnu-library/4.4/vitae/vitae.csl

---



---

# About me
\newcommand\tab[1][1cm]{\hspace*{#1}}
\begingroup
\fontsize{10}{12}\selectfont
\tab[1cm]I'm a computational biologist fascinated by understanding the molecular biodiversity and mechanisms. I have experience with workflow development in Python and R, molecular dynamics simulations, and immunogenetics. Currently, my work focuses on workflow development for data analysis from molecular dynamics simulations.  I’m an intern researcher at the Computational Biocatalysis Group at LNBR/CNPEM in Brazil, where I work with specific Carbohydrate-Active Enzymes (CAZymes) for microbiome and structural biology research projects using biomolecular modeling data and biochemical interpretation. I have been delivering valuable insights throughout research projects and dedicating myself to developing useful tools to enhance group performance. In my free time, I enjoy drawing molecular scientific illustrations, read about Latin culture and cooking plant-based recipes. 

\endgroup

# Education

\begin{cventries}
	\cventry{Bachelor in Biotechnology}{Federal University of Latin-American Integration (UNILA)}{Foz do Iguaçu, Brazil}{2018-2024}{}\vspace{-4.0mm}
	\cventry{Electrical Technician}{Liberato Salzano Vieira da Cunha Foudation}{Novo Hamburgo, Brazil}{2013-2017}{}\vspace{-4.0mm}
\end{cventries}

# Experience

\begin{cventries}
	\cventry{Computational Biocatalysis Intern}{Brazilian Center for Research in Energy and Materials (CNPEM)}{Campinas, São Paulo}{Feb 2023 - Present}{\begin{cvitems}
\item I have performed data analysis of several molecular dynamics simulations and large-scale molecular docking, primarily using Python and Shell scripting compatible with Slurm HPC environment. The main goal are new insights about molecular mechanisms from new Carbohydrate-Active enzyme families, which benefits the experimental team for mutation screening and activity assays.
\item \textbf{|} Molecular Dynamics and Structural Bioinformatics \textbf{|} Gromacs \textbf{|} Slurm/HPC \textbf{|} Python/MDAnalysis \textbf{|} Autodock Vina \textbf{|} Conda/Bioconda \textbf{|} Pymol/VMD \textbf{|}
\end{cvitems}}
\end{cventries}

\begin{cventries}
	\cventry{Freelancer - Jr Data Analyst}{SporeData Inc.}{Remote}{Oct 2022 - Feb 2023}{\begin{cvitems}
\item Worked as a freelancer on clinical data projects, performing ETL, cohort generation, data visualization, clinical study reports, and statistical analyses.
\item \textbf{|} Clinical Data \textbf{|} R \textbf{|} SparkR \textbf{|} RMarkdown \textbf{|} Cloud/N3C \textbf{|}
\end{cvitems}}
\end{cventries}

\begin{cventries}
	\cventry{Undergraduate Researcher}{Medical Genetics Lab - UNILA}{Foz do Iguaçu, Brazil}{Aug 2020 - Mar 2023}{\begin{cvitems}
\item I performed DNA sequencing and HLA-B analysis of COVID-19 patients, gathering clinical and genomic data, along with HLA-epitope binding affinities from computational predictions, to study the mutational landscape of SARS-CoV-2 variants in Foz do Iguaçu. I worked with a Nextflow pipeline to collect, transform, and visualize missense mutations affecting HLA epitopes based on machine learning tools. The result is an easy-to-use pipeline that contributes to genomic surveillance approaches for a Brazilian border research group. \href{https://github.com/chagas98/sarscovHLAFoz}{(\underline{Details})}
\item \textbf{|} Missense mutation analyses \textbf{|} GISAID \textbf{|} Nextflow \textbf{|} Bash \textbf{|} Python \textbf{|} R \textbf{|} Sanger Sequencing \textbf{|} netMHCpan4.1 \textbf{|}
\end{cvitems}}
\end{cventries}

\begin{cventries}
	\cventry{DryLab Coordinator}{International Genetically Engineered Machine Competition (iGEM)}{Paris, France}{Jan 2020 - Dez 2021}{\begin{cvitems}
\item As the Head of DryLab at the iGEM UNILA LatAm Team, I led the development of a kinetic model with parameter optimization. We used an Ordinary Differential Equations (ODEs) framework coupled with sensitivity analysis to evaluate the performance of our synthetic genetic circuit. This model provided valuable insights to the experimental team, helping them design experiments and identify key parameters to improve system performance. Also, we built the Carpincho Toehold Generator, specialized in the design of Toeholds for miRNAs detection, using Python and NUPACK API. \href{https://github.com/chagas98/iGEM2021_Modeling}{(\underline{Details})}
\item \textbf{|} System biology \textbf{|} Python \textbf{|} R \textbf{|}
\end{cvitems}}
\end{cventries}

\begin{cventries}
	\cventry{Undergraduate Researcher}{Microbiology and Biochemical Lab - UNILA}{Foz do Iguaçu, Brazil}{Jun 2018 - Dez 2019}{\begin{cvitems}
\item Worked with bacterial nanocellulose production for electrochemical applications, establishing bacteria cultivation, biofilm extraction and purification protocols. Additionally, conducted a literature review bridging the gap between bacterial nanocellulose and its electrochemical applications.
\item \textbf{|} Laboratory practices \textbf{|} Literature review \textbf{|} Microbiology lab protocols \textbf{|}
\end{cvitems}}
\end{cventries}


# Skills

\begin{cventries}
	\cventry{Python | Bash | R}{Programming}{}{}{}\vspace{-4.0mm}
\end{cventries}\begin{cventries}
	\cventry{Nextflow}{Workflow}{}{}{}\vspace{-4.0mm}
\end{cventries}\begin{cventries}
	\cventry{HTML | RMarkdown}{Markup}{}{}{}\vspace{-4.0mm}
\end{cventries}\begin{cventries}
	\cventry{Inkscape | Blender}{Scientific Illustration}{}{}{}\vspace{-4.0mm}
\end{cventries}\begin{cventries}
	\cventry{Blender/Molecular Nodes | Pymol | VMD | Gromacs}{Biomolecular Tools}{}{}{}\vspace{-4.0mm}
\end{cventries}\begin{cventries}
	\cventry{Git | Github | Slurm | R/Shiny}{Tools}{}{}{}\vspace{-4.0mm}
\end{cventries}

# Accomplishments

\begin{cvhonors}
	\cvhonor{}{1st Place. Honorable Mention in Health Science at XI EICTI, UNILA}{}{2022}
	\cvhonor{}{Gold Medal at International Genetically Engineering Machine Competition - iGEM}{}{2021}
	\cvhonor{}{Best Public Choice Project at the V Biological Machines Engineering Summer Program - UFMG}{}{2020}
	\cvhonor{}{Best Presentation Award at the V Biological Machine Engineering Summer Program - UFMG}{}{2020}
	\cvhonor{}{1st Place. Honorable Mention in Technology and Production at VII SEUNI - UNILA}{}{2019}
	\cvhonor{}{Honorable Mention for Socially Relevant Project at International Conference MOSTRATEC, UNESCO.}{}{2016}
\end{cvhonors}

# Publications

\begingroup
\fontsize{10}{12}\selectfont

* **De Assis**, S.C., Morgado, D.L., Scheidt, D.T., De Souza, S.S., Cavallari, M.R., Ando , O.H.J., Carrilho, E. (2023). Review of Bacterial Nanocellulose-Based Electrochemical Biosensors: Functionalization, Challenges, and Future Perspectives. *MDPI Biosensors*, 13. [https://doi.org/10.3390/bios13010142](https://doi.org/10.3390/bios13010142)


\endgroup

